presentation/common/src/presentation-common/KeySet.ts

```diff
@@ -176,8 +180,11 @@ export class KeySet {
   private addKeySetJSON(keyset: Readonly<KeySetJSON>): void {
     for (const key of keyset.nodeKeys)
       this._nodeKeys.add(JSON.stringify(key));
-    for (const entry of keyset.instanceKeys)
-      this._instanceKeys.set(entry["0"], new Set(entry["1"]));
+    for (const entry of keyset.instanceKeys) {
+      this._instanceKeys.set(entry["0"].toLowerCase(), new Set(entry["1"]));
```

Comment:
Could convert the class name to lower case only once

Reply:
Fixed it